business_requirement/tests/test_br.py

```diff
@@ -113,19 +113,21 @@ def test_create_sequance(self):
             'name': '/',
             'description': 'test',
         }
-        self.br.create(br_vals)
+        res = self.br.create(br_vals)
+        self.assertTrue(res.name)
```

Comment:
@victormartinelicocorp 
Sequance is automated created in create method i hade tested but it's not Equal.
`vals['name'] = self.env['ir.sequence'].next_by_code(
                'business.requirement')`